lib/plugins/aws/remove/lib/stack.js

```diff
@@ -26,8 +27,47 @@ module.exports = {
       .then(() => cfData);
   },
 
+  disassociateUsagePlan() {
```

Comment:
I can give it another try later, but I'm afraid that the order of the removal is still wrong.